linode.yml

```diff
@@ -3,10 +3,12 @@
 # Include your site.yml or just use the ceph-ansible sample:
 - include: "{{ lookup('env','CEPH_ANSIBLE') }}/site.yml.sample"
   vars:
-    cluster_network: 192.168.0.0/16
+    monitor_address_block: 192.168.128.0/17
+    public_network: 192.168.128.0/17
```

Comment:
ditto

Reply:
I guess we shouldn't need monitor_address_block if network interface for monitors is always "eth0", but it just seems stupid to specify monitor IPs by network interface name when it is implicit in public_network var.

Reply:
The network interface `eth0` on Linode has multiple IP addresses so we can't just specify the network interface to ceph-ansible. Linode is the only use-case for `monitor_address_block` for exactly this reason.

Reply:
we need rgw_address_block as well, am going to start over on this PR.